code/game/gamemodes/cult/runes.dm

```diff
@@ -1,1053 +1,982 @@
-				if(ticker.mode.name == "cult")
-					if(M.mind == ticker.mode.sacrifice_target)
-						for(var/mob/living/carbon/human/cultist in cultsinrange)
-							cultist << "<span class='h2.boldannounce'>The Chosen One!! <BR>KILL THE CHOSEN ONE!!! </span>"
-				return 0
-		else
-			for(var/mob/living/carbon/human/cultist in cultsinrange)
-				cultist << "<span class='warning'>You need more brothers to overcome their lies and make them see Truth. </span>"
-	return fizzle()
-
-
-///Summon Nar-Sie: Summons the avatar of Nar-Sie to wreak havoc on the station. Will turn walls and floors to cult version and make anyone it touches into a harvester.
-
-
-/obj/effect/rune/proc/tearreality()
-	var/list/mob/living/cultist_count = list()
-	var/mob/living/user = usr
-	user.say("Tok-lyr rqa'nap g[pick("'","`")]lt-ulotf!")
-	for(var/mob/M in range(1,src))
-		if(iscultist(M) && !M.stat)
-			cultist_count += M
-	if(cultist_count.len >= 9)
-		if(ticker.mode.name == "cult")
-			var/datum/game_mode/cult/cultmode = ticker.mode
-			if(!("eldergod" in cultmode.cult_objectives))
-				message_admins("[usr.real_name]([usr.ckey]) tried to summon a god when she didn't want to come out to play.")	// Admin alert because you *KNOW* dickbutts are going to abuse this.
-				for(var/mob/M in cultist_count)
-					M.reagents.add_reagent("hell_water", 10)
-					M << "<span class='h2.userdanger'>YOUR SOUL BURNS WITH YOUR ARROGANCE!!!</span>"
-				return
-			else
-				for(var/obj_count=1, obj_count <= cultmode.cult_objectives.len, obj_count++)
-					if(cultmode.cult_objectives[obj_count] == "sacrifice")
-						if(cultmode.sacrifice_target)
-							if(!(cultmode.sacrifice_target in sacrificed))
-								for(var/mob/M in cultist_count)
-									M << "<span class='h2.userdanger'>Nar-sie refuses to be summoned while the sacrifice isn't complete.</span>"
-								return
-				cultmode.eldergod = 0
-		var/narsie_type = /obj/singularity/narsie/large
-		// Moves narsie if she was already summoned.
-		var/obj/her = locate(narsie_type, SSobj.processing)
-		if(her)
-			her.loc = get_turf(src)
-			return
-		// Otherwise...
-		new narsie_type(src.loc) // Summon her!
-		qdel(src) 	// Stops cultists from spamming the rune to summon narsie more than once.
-					// Might actually be wise to straight up del() this
+	var/obj/effect/rune/selected_rune = pick(potential_runes)
+	var/mob/living/target
+
+	var/list/targets = list()
+	for(var/mob/living/L in get_turf(src))
+		if(L != user)
+			targets.Add(L)
+	if(!targets.len)
+		user << "<span class='warning'>There are no targets standing on the rune!</span>"
+		fail_invoke()
+		log_game("Teleport Other rune failed - no targets on rune")
 		return
+	if(targets.len > 1)
+		target = input(user, "Choose a person to teleport.", "Rite of Forced Translocation") as null|anything in targets - user
+		if(!target)
+			fail_invoke()
+			return
 	else
-		return fizzle()
-
-
-///EMP: Lets out a large EMP blast
+		target = targets[targets.len]

```

Comment:
did you mean ```target = targets[target]```?
because right now you're disregarding what the user chose, and always picking the last person in the list.